api/v1/swagger.yaml

```diff
@@ -4484,33 +4489,40 @@ paths:
           schema:
             type: object
             description: >
-              Credentials of person who interacts with the Refocus system.
+              Name of token.
             properties:
-              email:
+              name:
                 type: string
                 maxLength: 254
                 description: >
-                  The user's email address.
-              password:
-                type: string
-                format: password
-                description: >
-                  The user's password.
+                  The name of token. Token name should be unique for a user.
```

Comment:
A name for your new token. Token name must be unique for a user. Best practice is to select a name which reminds you of how you intend to use the token, e.g. "PushSamplesFromSystemXyz".